app/models/user.rb

```diff
@@ -8,6 +8,8 @@ class User < ApplicationRecord
 
   # add validations and rspec
   validates :nickname, presence: true, uniqueness: true
+  validates :first_name, presence: true
+  validates :last_name, presence: true
```

Comment:
no need for these validations, we want to keep privacy so they don't have to put in their real names

Reply:
Kk done.